orchagent/copporch.cpp

```diff
@@ -73,6 +73,8 @@ static map<string, sai_hostif_trap_type_t> trap_id_map = {
     {"udld", SAI_HOSTIF_TRAP_TYPE_UDLD},
     {"bfd", SAI_HOSTIF_TRAP_TYPE_BFD},
     {"bfdv6", SAI_HOSTIF_TRAP_TYPE_BFDV6}
+    {"src_nat_miss", SAI_HOSTIF_TRAP_TYPE_SNAT_MISS},
```

Comment:
Can we make sure that these trap ids are created only on platform which support NAT

Reply:
Done based on the feature check.